snappi_ixnetwork/ixnetworkapi.py

```diff
@@ -154,31 +166,47 @@ def set_config(self, config):
             with Timer(self, 'Flows configuration'):
                 self.traffic_item.config()
         self._running_config = self._config
+        self._apply_change()
         return self._request_detail()
 
-    def set_transmit_state(self, flow_transmit_state):
+    def set_transmit_state(self, payload):
         """Set the transmit state of flows
         """
-        if isinstance(flow_transmit_state, str) is True:
-            flow_transmit_state = self.transmit_state().deserialize(
-                                    flow_transmit_state)
+        if isinstance(payload, (type(self.transmit_state()),
```

Comment:
store transmit_state() in var here as well

Reply:
added